src/wazuh_db/wdb.h

```diff
@@ -389,6 +389,20 @@ int wdb_sca_policy_sha256(wdb_t * wdb, char *id, char * output);
  */
 void wdb_free_agent_info_data(agent_info_data *agent_data);
 
+/**
+ * @brief Function to parse a chunk response that containing the status of the query and a json array.
```

Comment:
Typo... `that contains`, or in any case just `containing`.

Reply:
Done